static/app/components/searchQueryBuilder/index.tsx

```diff
@@ -102,6 +102,10 @@ export interface SearchQueryBuilderProps {
    */
   onSearch?: (query: string, state: CallbackSearchState) => void;
   placeholder?: string;
+  /**
+   * If provided, will render the combobox popovers into the given element.
```

Comment:
tiny nit/suggestion: might be helpful to describe that this is useful when the search input in rendered in a scroll container, just so future devs can better understand the purpose of this prop